reset_user_access_right/views/res_users_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+	   <record id="view_users_form" model="ir.ui.view">
+            <field name="name">res.users.form (reset_user_access_right)</field>
+            <field name="model">res.users</field>
+            <field name="inherit_id" ref="base.view_users_form" />
+            <field name="arch" type="xml">
+                <xpath expr="//group[@string='Allowed Companies']" position="before">
+                    <button name="reset_access_right" string="Reset Access Right" type="object" />
```

Comment:
Can you add a confirm="This will reset the user's access rights to the minimum configuration. Are you sure?" text?